packages/plans-grid-next/src/components/test/tooltip.tsx

```diff
@@ -0,0 +1,78 @@
+/**
+ * @jest-environment jsdom
+ */
+
+import { render, screen, waitFor } from '@testing-library/react';
+import React from 'react';
+import Tooltip from '../shared/tooltip';
+
+const tooltipId = 'tooltip-123';
+const tooltipAnchor = 'Hover me';
+const tooltipHoverText = 'Wow, that was fun';
+
+describe( 'Tooltip', () => {
+	it( 'should display the anchor', () => {
```

Comment:
What exactly are we testing here? 

- under what conditions would this fail
- how likely is it to be overlooked

Reply:
This test will fail if the anchor (the element that is displayed in order for the tooltip to function) is ever completely removed. I've [updated](https://github.com/Automattic/wp-calypso/pull/93518/commits/dcb29efa497ddebdf2603328665235311407dfa6) the test descriptions to improve clarity.

In terms of 'how likely' this is -- I think that's a subjective question. As in, you will get a different answer depending on who you ask. From my perspective, testing each possible state for a component (however unlikely) leads to a positive outcome (less bugs).

Can you elaborate on your perspective? With this test in place, what are the negative outcomes?